src/Escaper.php

```diff
@@ -67,20 +65,23 @@ class Escaper
      * Static Matcher which escapes characters for HTML Attribute contexts
      *
      * @var callable
+     * @psalm-var callable(array<array-key, string>):string
```

Comment:
Seems like this callback expects `/** @psalm-var list<string> $matches */`? :thinking: 

Same applies to the other callbacks?

Reply:
Yes, they all go to `preg_replace_callback` - the problem is annotating the assignment `$this->callback = [$this, 'some-method']` - As you see, I've added the property type and updated the class method parameters. What psalm-foo can be used to annotate the array callable without assigning to a temporary var and labelling that with `psalm-var`??

Reply:
> What psalm-foo can be used to annotate the array callable without assigning to a temporary var and labelling that with `psalm-var`??

Works: better than taking a double callback impact on something as hot as the escaper